Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:10-alpine as deps
+
+COPY package.json yarn.lock ./
+
+RUN yarn && \
```

Comment:
Would just `yarn --modules-folder node_modules_prod` on its own work? I think `yarn` and `yarn install` do the same thing

Reply:
that is creating a dependencies image for us to use on the next two steps. We need dev dependencies to run tests, and we only need production dependencies to run production. This approach allows us to cache both for use on demand and speed up our builds considerably.

Reply:
Sweet